WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+
+/// SiteAddressViewController: log in by Site Address.
+///
+class SiteAddressViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    
+
+    /*
+    // MARK: - Navigation
```

Comment:
Nitpick: I know this is a WIP, but it'd be nice if all the commented out boilerplate stuff could be removed before merging.